jbmc/src/Makefile

```diff
@@ -40,11 +41,33 @@ generated_files: $(patsubst %, %_generated_files, $(DIRS))
 # cleaning
 
 .PHONY: clean
-clean: $(patsubst %, %_clean, $(DIRS)) cprover_clean
+clean: $(patsubst %, %_clean, $(DIRS)) cprover_clean dist_clean
 
 $(patsubst %, %_clean, $(DIRS)):
 	$(MAKE) $(MAKEARGS) -C $(patsubst %_clean, %, $@) clean ; \
 
 .PHONY: cprover_clean
 cprover_clean:
 	$(MAKE) $(MAKEARGS) -C $(CPROVER_DIR)/src clean
+
+.PHONY: dist_clean
+dist_clean:
+	rm -rf $(ROOT)dist
+
+# extended JBMC models download, for your convenience
+java-models-library-download:
+	@echo "Downloading java models library"
+	@curl -L https://github.com/diffblue/java-models-library/archive/master.zip > java-models-library.zip
```

Comment:
I would have use the `/tmp` folder as a temporary folder to store the zip file, but that's not very important.

Reply:
I took this approach as its the one being used for minisat, for the sake of consistency, will leave it as is.

Reply:
The minisat download uses (by default) `lwp-download`, which doesn't pick up the OS certificate chain required for HTTPS. I'm tempted to suggest to get rid of `lwp-download` instead.